### PR TITLE
fusermount: add page

### PR DESCRIPTION
**Pull request by marchersimon**

### CODE REVIEW

File: pages/common/fusermount.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Unmount a FUSE filesystem:
+
+`fusermount -u {{path/to/mount_point}}`
+
+- Unmount a FUSE filesystem, as soon as it's not busy anymore:
+
+`fusermount -z {{path/to/mount_point}}`
+
+- Print the version of `fusermount` and exit:
```
Comment (navarroaxel) [inline pages/common/fusermount.md:14]:
```suggestion
- Display version:
```

File: pages/common/fusermount.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# fusermount
+
+> Mount and unmount FUSE filesystems.
```
Comment (navarroaxel) [inline pages/common/fusermount.md:3]: I can´t find an example for mount using this command :thinking:
Maybe we should update the command description only for `unmount`.

Comment (marchersimon) [reply pages/common/fusermount.md:3]: There is a `-o` option which expects mount options, so it should at least be possible to mount filesystems. But I couldn't find those options anywhere.

Comment (marchersimon) [reply pages/common/fusermount.md:3]: Maybe those are the same options for `mount`, but that's just an asumption.

File: pages/common/fusermount.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://man.archlinux.org/man/fusermount.1>.
+
+- Unmount a FUSE filesystem:
+
+`fusermount -u {{path/to/mount_point}}`
+
+- Unmount a FUSE filesystem, as soon as it's not busy anymore:
```
Comment (bl-ue) [inline pages/common/fusermount.md:10]:
```suggestion
- Unmount a FUSE filesystem as soon as it becomes unused:
```

File: pages/common/fusermount.md (hunk 2 above)
Comment (bl-ue) [reply pages/common/fusermount.md:3]: The page doesn't need to have all possible examples, so we can leave it for someone else to pick up later.

Comment (bl-ue) [reply pages/common/fusermount.md:3]: I think so.  It might even prompt a reader to say "why not?" and then fix it. 🤷🏻

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**